create-cookie-keys.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+rm -rf cookie-encryption 2>/dev/null
+mkdir cookie-encryption
+cd cookie-encryption
+
+#
+# The session cookie is encrypted using the system's symmetric key
+#
+echo 'Creating keys for cookie encryption ...'
+export SYMMETRIC_KEY=$(openssl rand 32 | xxd -p -c 64)
+
+#
+# Generate proxy cookie keys
+#
+export TH_COOKIE_KEY_PASS='Password1'
+openssl ecparam -name prime256v1 -genkey -noout -out private-key.pem
+openssl ec -in private-key.pem -pubout -out public-key.pem
+openssl pkcs8 -topk8 -in private-key.pem -out private-key.pkcs8 -passout pass:"$TH_COOKIE_KEY_PASS"
+rm private-key.pem
+if [ $? -ne 0 ]; then
+  echo 'Problem encountered generating cookie encryption keys'
+  exit 1
+fi
+cd ..
+
+#
+# Get the proxy private key into a single line format to supply to API gateways
+#
+export TH_COOKIE_KEY="$(awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' ./cookie-encryption/private-key.pkcs8)"
```

Comment:
The multiline certificate files are a bit awkward to handle from a scripting viewpoint. This solution is quite common if you search on a term like the following, though it is more cryptic than I'd like:

- multiline cert to single line